exercise/static/exercise/language_link.js

```diff
@@ -0,0 +1,19 @@
+function set_language_links(set_language_url) {
+  $('a.change-language').each(function (i, elem) {
+    const $elem = $(elem);
+    const target_lang = $elem.attr('data-target-lang');
```

Comment:
Was there some difference to `.data('target-lang')`?

Reply:
In this case, there shouldn't be any. I generally prefer `.attr()` because in more complicated cases `.data()` involves some black magic which IMO can make the code hard to read.